dace/sdfg/state.py

```diff
@@ -1097,10 +1110,27 @@ def add_mapped_tasklet(
         """
         map_name = name + "_map"
         debuginfo = _getdebuginfo(debuginfo)
+
+        # Create appropriate dictionaries from inputs
+        if isinstance(inputs, set):
+            tinputs = {k: None for k in inputs}
+        else:
+            tinputs = {
+                k: self.parent.arrays[v.data].dtype
```

Comment:
The type hint seems to say that v is a typeclass, but here it seems to be a memlet

Reply:
The type hint says it's only a dictionary to a memlet. Removed the Set[str] case.